tools/migrateInheritSteward/migrate.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import json
+import os
+import subprocess
+import time
+import uuid
+
+from data_repo_client import (
+    Configuration,
+    ApiClient,
+    DatasetsApi,
+    JobsApi,
+)
+
+
+class Clients:
+    def __init__(self, host):
+        config = Configuration()
+        config.host = host
+        token_output = subprocess.run(
+            ["gcloud", "auth", "print-access-token"], capture_output=True
+        )
+        config.access_token = token_output.stdout.decode("UTF-8").strip()
+        self.api_client = ApiClient(configuration=config)
+
+        self.datasets_api = DatasetsApi(api_client=self.api_client)
+        self.jobs_api = JobsApi(api_client=self.api_client)
+
+
+def wait_for_job(clients, job_model):
+    result = clients.jobs_api.retrieve_job(job_model.id)
+    while True:
+        if result is None or result.job_status == "running":
+            time.sleep(10)
+            print(f"Waiting for job {job_model.id} to finish")
+            result = clients.jobs_api.retrieve_job(job_model.id)
+        elif result.job_status == "failed":
+            print(job_model.id)
```

Comment:
Is this for debugging? Or should it have a Failed message?
```suggestion
            print(f"Job {job_model.id} failed")
```